src/ErrorHandler.php

```diff
@@ -41,10 +41,15 @@ final class ErrorHandler
     private $previousExceptionHandler;
 
     /**
-     * @var int The errors that will be catched by the error handler
+     * @var int The errors that will be caught by the error handler
      */
     private $capturedErrors = E_ALL;
 
+    /**
+     * @var int The last caught error type
+     */
+    private $lastErrorType = 0;
```

Comment:
Why using a property and not passing it as every invocation of `handleException`?